### PR TITLE
Renamed cpp to c and cpp2 to cpp

### PR DESCRIPTION
**Pull request by TwoOfTwelve**
Renamed cpp to c and cpp2 to cpp as discussed.

### CODE REVIEW

**Comment by Kr0nox** [comment]
And update the README of the now cpp (formerly cpp2) module, since it still mentions the module as cpp2

File: report-viewer/src/model/Language.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@
 enum ParserLanguage {
   JAVA = 'Javac based AST plugin',
   PYTHON = 'Python3 Parser',
-  CPP = 'C/C++ Scanner [basic markup]',
-  CPP2 = 'C/C++ Parser',
+  C = 'C(C++) Scanner [basic markup]',
+  CPP = 'C++(C) Parser',
```
Comment (tsaglam) [inline report-viewer/src/model/Language.ts:8]:
```suggestion
  C = 'C Scanner',
  CPP = 'C++ Parser',
```

File: languages/c/src/main/java/de/jplag/c/CLanguage.java [hunk 2]
```diff
@@ -26,7 +27,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return "C/C++ Scanner [basic markup]";
+        return "C(C++) Scanner [basic markup]";
```
Comment (tsaglam) [inline languages/c/src/main/java/de/jplag/c/CLanguage.java:30]:
```suggestion
        return "C Scanner";
```

File: languages/cpp/src/main/java/de/jplag/cpp/CPPLanguage.java [hunk 3]
```diff
@@ -23,7 +23,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return "C/C++ Parser";
+        return "C++(C) Parser";
```
Comment (tsaglam) [inline languages/cpp/src/main/java/de/jplag/cpp/CPPLanguage.java:26]:
```suggestion
        return "C++ Parser";
```

**Review by tsaglam — CHANGES_REQUESTED** [review]
Please also update the language table in the main readme, add a readme to c, update the cpp readme, and check if the wiki needs to be updated.

**Comment by tsaglam** [comment]
@TwoOfTwelve since sonar is back, we have a few issues here. Also don't forget my comments regarding the readme files.

File: README.md [hunk 4]
```diff
@@ -146,14 +146,13 @@ Clustering
...
 Commands:
+  c
   cpp
-  cpp2
   csharp
   emf
   emf-model
   go
   java
-  javascript
```
Comment (Kr0nox) [inline README.md:156]: This line should not be removed

File: languages/cpp/README.md
Comment (Kr0nox) [inline languages/cpp/README.md:1]: > And update the README of the now cpp (formerly cpp2) module, since it still mentions the module as cpp2

**Review by Kr0nox — CHANGES_REQUESTED** [review]

**Review by tsaglam — APPROVED** [review]

**Comment by tsaglam** [comment]
@Kr0nox ready from your side?

**Review by Kr0nox — APPROVED** [review]

**Pull request merged**